### DISCUSSION TITLE
Does this work as an integration with FIREBASE

### DISCUSSION THREAD

**Discussion by dummyastrick12** [saran-surya/email_auth_node#48] [Q&A] [answered]
is this work on firebase if yes then sendme the steps

**Comment by saran-surya** [comment, marked as answer]
> is this work on firebase if yes then sendme the steps

Hi @dummyastrick12
It could help if you could learn to ask questions politely / clearly in the first place. 

And no, firebase already has its own authentication procedures, so I kindly request you to check them out. It is also now made simpler with latest methods.

Thanks
